### ISSUE TITLE
manuscript

### ISSUE THREAD

**Issue by SeascapeScience** [BigelowLab/jellyscape#6]
I'm toying with the idea I mentioned of drafting up a manuscript that answers some basic questions using these time series. Should I make a new directory "manuscript"... or would you organize differently?

**Comment by btupper** [comment]
Let's start with that - you are doing it in RMarkdown?

**Comment by SeascapeScience** [comment]
I think we can do R markdown. I'll start with that.

**Comment by SeascapeScience** [comment]
Okay, it's there as a markdown document. We can add coded figures as they get finalized.

**Comment by btupper** [comment]
FYI - there is a 'setup.R' to source.  You may need to do it ala `source("../setup.R")` but we may need to experiment.

**Comment by SeascapeScience** [comment]
@btupper Here is something we could use for the manuscript.

Q: Are populations increasing?

Answering requires multiple steps.
1. Subdivide into regions. We could either do this as broad polygons (I'd suggest three: Georges Bank, Gulf of Maine, and Mid Atlantic), or we could break into something like 1-degree rectangles.
2. Average the data for each species into these regions and convert resulting time series into monthly anomalies.
3.  Test each time series for statistically significant increase (we'll have to decide on a test-- there are lots of choices)
4. Plot back on map for nice viz

Notes: 
 - the data are probably log-normal, so we should use data on a log scale.
 - we could also look at something like proportion of presences

Anyway, I think this would be enough for the core of a fairly simple but valuable paper. Whenever you feel like tackling these figures, let me know if you have questions.

**Comment by btupper** [comment]
Yes, looks easy.  I might use hexagons for this part since they tile into regional sub-groups easily, are easy to store and follow the shoreline nicely. But if that isn't appealing I can switch to a regular grid.  

Log-scale abundance - roger that.

**Comment by SeascapeScience** [comment]
Hexagons could be good. If they're too small, you might find not enough data to detect trends. When I did this with ecomon calanus data, I found 1 degree to be about the right size.

**Comment by btupper** [comment]
OK - I'll make them biggish.  Iz good?

![Image](https://github.com/user-attachments/assets/71ff8c5d-b6f4-431d-8254-a96f346304cd)

**Comment by btupper** [comment]
[This is a decadal anomaly series](https://bigelowlab.github.io/jellyscape/07_changes.html#decadal-abundance) for total abundance (count/10m^2) as a departure from the full mean.  Abundances are transformed with `log1p(abundance)` as the data are read form disk.  We can show the color bar in 'real' unlogged units but it's probably not worrying about that until the end. 

If you squint at, say SE of Yarmouth NS you'll see some empty bins.  That surprised me as the bins were defined by ALL of the points, and presumably the total abundance will be non-NA in each cell.  But, of course, I forgot that each mini-map is for a decade so not every cell gets a value in every decade.  So we are good!  Per species is next.

**Comment by SeascapeScience** [comment]
Looks good. We also need to do anomalies off the climatology because the sampling is not the same across decades. In other words, the January mean should be subtracted from each January year (within each hexagon), and so on for Feb-Dec.
